src/ctl/cart_ctl.c

```diff
@@ -651,31 +604,13 @@ main(int argc, char **argv)
 	int		rc = 0;
 
 	rc = parse_args(argc, argv);
-	if (rc != 0) {
-		D_ERROR("parse_args() failed. rc %d\n", rc);
-		D_GOTO(out, rc);
-	}
-	rc = ctl_init();
-	if (rc != 0) {
-		D_ERROR("ctl_init() failed, rc %d\n", rc);
-		D_GOTO(out, rc);
-	}
-
-	rc = ctl_issue_cmd();
+	D_ASSERTF(rc == 0, "parse_args() failed. rc %d\n", rc);
 
-	if (rc != 0) {
-		D_ERROR("Command '%s' failed with rc=%d\n",
-			cmd2str(ctl_gdata.cg_cmd_code), rc);
-		D_GOTO(out, rc);
-	}
+	/* rank, pid, is_server, num_attach_retries, do_assert */
+	tc_test_init(0, getpid(), 0, 20, 0);
```

Comment:
for is_server/do_assert its better to pass bool type -- true/false; easier to read